dashboard/config/locales/en.yml

```diff
@@ -421,14 +421,12 @@ en:
     autoplay_option: 'Autoplay videos'
     download: 'Download Video'
   compatability:
-    working:
-      header: "We are working on support for your browser."
-      message: "Code.org uses modern web technology. This makes the site faster to use. Your browser doesn't support all the features required, and you may experience issues while using this part of the site. Please help us by <a href='%{report_issue_url}'>reporting any issues you find</a>. You can also <a href='%{supported_browsers_url}'>upgrade your browser to a supported version</a>."
-      hide_this_warning: "Hide this warning"
     upgrade:
-      header: "Please upgrade your browser."
-      message: "Code.org uses modern web technology. This makes the site faster to use. Unfortunately your browser doesn't support all the features required to run this part of the site.<br/><br/>Install a browser on our supported browsers list to continue. You can <a href='#'>try viewing the page</a>, but expect functionality to be broken."
-      button: "Take me to the list of supported browsers"
+      header: "Your browser is not supported, please upgrade your browser."
```

Comment:
Not sure if comma is quite appropriate here.  Myabe "Your browser is not supported.  Please upgrade."?